components/templates/MDContent.js

```diff
@@ -0,0 +1,20 @@
+import Head from 'next/head';
+import MDRenderer from '../molecules/MDRenderer';
+import ContentContainer from '../organisms/ContentContainer';
+
+export default function MDContent(props) {
+    return (
+        <>
+            <Head>
+                <title>{props.pattern.data.name}</title>
+                <meta name="description" content={props.pattern.data["short-description"]} />
+            </Head>
+            <ContentContainer>
+                <h1>{props.pattern.data.name}</h1>
+                { props.children }
+                <MDRenderer markdown={props.pattern.markdown} />
+                <div>{props.t["Date Modified"]}: {props.pattern.data["date-modified"]}</div>
+            </ContentContainer>
+        </>     
+    )
+}
```

Comment:
Have a few comments for this one:

- No styling is imported, for example `H1` elements have a red line under them and content spacing is adjusted. These styles should be copied from the current pattern page and implemented here.
- I see what you're doing with `{ props.children }` and Pattern.js. Templates shouldn't import templates, this is why I was only having 1 template for both pages before. If you want them separate they should be 2 independent templates that are effectively clones of one another. The other option is to make the Pattern page into an organism and pass that into MDContent via [slug].js
- MDContent has been made into a generic page, prop names should be adjusted so we don't have `props.pattern` anymore
- It's recommended to use `<React.Fragment>` over <>. Apparently some babel versions have issues with this. Not a big deal but something to consider.

Reply:
>No styling is imported, for example H1 elements have a red line under them and content spacing is adjusted. These styles should be copied from the current pattern page and implemented here.

Fixed, but I added to the global css so it never gets left out.

>I see what you're doing with { props.children } and Pattern.js. Templates

addressing below

>MDContent has been made into a generic page

my plan was to do this in a separate PR to keep things cleaner

><React.Fragment> over <>

noted will leave it for now, can change if it causes us issues - seems to be common practice not to...

Reply:
Adding the CSS to globals prevents our components from being easily imported into other projects. Global styling should be left for basic things such as font-family and default font sizes. Everything else should be in a module for easier code re-use and to keep things cleaner overall.